src/s3-helper.js

```
@@ -67,7 +66,6 @@ export async function deleteAllObjects(s3, bucketName) {
   const oldObjects = await listAllObjects({ s3, bucketName });
   for (const obj of oldObjects) {
     console.log(`Deleting object ${obj.Key}`);
-    // eslint-disable-next-line no-await-in-loop
```

Nice! We've always been making an exception for this one as it was always an "informed decision" to ignore this error.

---

From now on we can be as uninformed as we want :rofl:

---

😂 Oblivion is bliss